sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/CheckDownloadTask.java

```diff
@@ -70,7 +70,7 @@
         mReleaseDetails = releaseDetails;
     }
 
-    @SuppressWarnings("deprecation")
+    @SuppressWarnings({"deprecation", "RedundantSuppression"})
```

Comment:
javac and in file analysis do not agree with the depreaction warnings, so the second suppress is needed to reconcile them